sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -283,6 +283,11 @@
   /** Controls if the `tracestate` header is attached to envelopes and HTTP client integrations. */
   private boolean traceSampling;
 
+  /**
+   * Contains a list of origins to which `sentry-trace` header should be sent in HTTP integrations.
+   */
+  private @NotNull List<String> tracingOrigins = new ArrayList<>();
```

Comment:
```suggestion
  private @NotNull List<String> tracingOrigins = new CopyOnWriteArrayList<>();
```

the reason is that people may iterate it and remove items on the go and if not thread-safe, the SDK crashes, its a client error, but it happened a few times.